packages/component/rollup.config.js

```diff
@@ -45,7 +45,9 @@ export default [
     external: Object.keys(globals),
     plugins: [
       babel(getBabelOptions({ useESModules: true })),
-      nodeResolve(),
+      nodeResolve({
+         mainFields: ['main'],
```

Comment:
build was triggering error because of something wrong in react-prerendered-component module version. I can invest some time fixing it if you are interested in the feature